src/components/layout/nav.js

```diff
@@ -230,19 +228,18 @@ export const Nav = () => {
                         <Button onClick={handleLogin} primary>
                             <span>{localize('Log in')}</span>
                         </Button>
-                        <SignupButton ref={button_ref} secondary onClick={toggleModal}>
-                            <span>{localize('Try for free')}</span>
-                        </SignupButton>
+                        <LocalizedLink to='/signup/'>
```

Comment:
There is a `LinkButton` that can be used here

Reply:
if I use `LinkButton` here for an unknown reason the calculation of translation gets messy and the login button will not show.